packages/actor-http-native/lib/Requester.ts

```diff
@@ -37,7 +37,11 @@ export default class Requester {
 
     // Unpacking headers object into a plain object
     const headersObject: any = {};
-    if (settings.headers) {
+    if (settings.headers && settings.headers._headers) {
```

Comment:
`_headers` is not part of the standard fetch API, so this should not be needed.
For what reason do you want this? If a test is failing, perhaps that test is wrong?

Reply:
there is one test that fails in `yarn run integration`: DBPedia TPF & SPARQL (https://comunica.github.io/manifest-ldf-tests/sparql/sparql-manifest.ttl#simple03) with following errorstack:
https://hastebin.com/kuhufoceku.sql

Reply:
Your hastebin is empty for me.

In any case, I assume _something_ is going wrong in another place in real requests, not corresponding to the test mocks. I would suggest debugging a bit to see why this difference exists.